packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js

```diff
@@ -125,6 +125,7 @@ export default class QuanticSearchInterface extends LightningElement {
   }
 
   disconnectedCallback() {
+    this.initialized = false;
```

Comment:
Ok so just to confirm, it was not calling the connectedCallback in `quanticSearchInterface` because it was calling the connectedCallback in the `HostedSearchInterface` which extends the `quanticSearchInterface` ?

And this works because in the `HostedSearchInterface` intialize method we call:


`  initializeHostedInterface = (engine) => {
    super.initialize(engine);
  };`
which calls the initialize in `quanticSearchInterface` and would do:



`    if (this.initialized) {
      return;
    }` 

and just return instead of initializing properly correct?

Reply:
Exactly yeah, the  `connectedCallback` of `quanticSearchnterface` is not called because this method, the `connectedCallback` of the `quanticSearchInterface` is being overridden by the   the `connectedCallback` of `hostedSearchInterface`.